src/blockchain/bulk-sync-augur-node-with-blockchain.ts

```diff
@@ -68,3 +75,29 @@ async function getHighestBlockNumber(augur: Augur): Promise<number> {
     });
   });
 }
+
+async function getIsEthNodeSyncing(augur: Augur): Promise<{
+  highestBlockNumber: number;
+  isEthNodeSyncing: boolean;
+}> {
+  const getHighestBlockNumberPromise = getHighestBlockNumber(augur);
+  const syncingPromise = new Promise<any>((resolve, reject) => {
+    augur.rpc.eth.syncing(null, (error, response: any) => {
```

Comment:
Can this be typed as boolean?

Reply:
No, because the result is an object if the node is syncing. https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_syncing

My need to model the response type explicitly was limited by fact that this change only cares if its `=== false`.